python/hail/representation/annotations.py

```diff
@@ -70,3 +71,29 @@ def get(self, item, default=None):
         :returns: Value of item if found, or default value if not.
         """
         return self._attrs.get(item, default)
+
+
+@typecheck(struct=Struct)
+def to_dict(struct):
```

Comment:
alright, I'll add it as an instance method. should there be a `recurse` boolean parameter to decide whether to do a deep conversion?